plugin/buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -0,0 +1,9 @@
+object Versions {
+    val kotlin = "1.2.30"
+    val ktlintPlugin = "3.1.0"
+    val gradlePublishPlugin = "0.9.10"
+    val androidPlugin = "3.0.1"
+    val kotlinNativePlugin = "0.6.1"
+    val semver = "1.0.0"
+    val gradleWrapper = "4.6"
+}
```

Comment:
❓ Some redundancy here? Any way to reduce it?

Reply:
I tried with `ln -s` as *plugin* is a separate Gradle project, but got `Redeclaration: Versions` error on compile. Can't figure out any other solution.

Reply:
Use a different name for the two classes? Call one `BuildSrcVersions` and the other something else?